ocean_lib/data_provider/data_service_provider.py

```diff
@@ -562,185 +447,6 @@ def compute_job_result_logs(
 
         return function_result
```

Comment:
Expected type `Dict[str, Any]` and function_result has `List[Dict[str, Any]]` type.

Reply:
changed